tests/ignite/metrics/test_ssim.py

```diff
@@ -9,6 +9,8 @@
 from ignite.exceptions import NotComputableError
 from ignite.metrics import SSIM
 
+from tests.ignite import cpu_and_maybe_cuda
```

Comment:
Isn't better to be something like `available_devices`?  This gives the feeling of actually importing cpu and maybe cuda!

Reply:
OK, let's rename it. Thanks for the feedback

Reply:
It could also be a fixture to be used just by declaring as argument instead of writing `@pytest.mark.parametrize...`.